packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -565,6 +565,7 @@ const MessageThreadStory = (args): JSX.Element => {
             />
           );
         }}
+        richTextEditor={args.richTextEditor}
```

Comment:
We probably need `/* @conditional-compile-remove(rich-text-editor) */` here as otherwise it will be visible in beta storybook builds (might be an issue during the next beta release)

Reply:
We can't do conditional compile in storybook though right? I believe this won't show up, as the property isn't there in stable/beta

Reply:
As discussed, we will remove this from the messageThread story until beta release